shared/lib-camel-connector/src/main/java/gov/va/vro/camel/RabbitMqCamelUtils.java

```diff
@@ -27,6 +27,14 @@ public static String rabbitmqTopicConsumerEndpoint(String exchangeName, String q
     return "rabbitmq:" + exchangeName + "?exchangeType=topic&queue=" + queueName;
   }
 
+  public static String rabbitmqFanoutConsumerEndpoint(String exchangeName, String queueName) {
+    return "rabbitmq:"
+        + exchangeName
+        + "?exchangeType=fanout&queue="
+        + queueName
+        + "&skipExchangeDeclare=true&skipQueueDeclare=true&skipQueueBind=true";
```

Comment:
Why skip declaring the exchange and queue?

Reply:
I figured that the xample-workflow as a client should not be responsible for ensuring that the queues and exchanges are created. Also, if the configuration (durable, auto-delete, etc.,) they attempt to create does not exactly match that of the already present queue/exchange, then it cannot connect.